src/pudl/analysis/service_territory.py

```diff
@@ -318,43 +240,47 @@ def get_balancing_authority_geometries(ids,
         generating visualizations.
 
     Args:
-        ids (iterable of ints): The set of balancing_authority_id_eia values for which
-            we are compiling geometries.
-        st_eia861 (pandas.DataFrame): The service_territory_eia861 table.
-        ba_assn_eia861 (pandas.DataFrame): The balancing_authority_assn_eia861 table.
+        ids (iterable of ints): A collection of EIA balancing authority IDs.
+        assn (pandas.DataFrame): Association table, relating ``report_date``,
+        ``state``, and ``utility_id_eia`` to each other, as well as the
+            column indicated by ``assn_col`` -- if it's not ``utility_id_eia``.
+        assn_col (str): Label of the dataframe column in ``assn`` that contains
+            the ID of the entities of interest. Should probably be either
+            ``balancing_authority_id_eia`` or ``utility_id_eia``.
+        st_eia861 (pandas.DataFrame): The EIA 861 Service Territory table.
```

Comment:
why not generate this inside of this function with a pudl_out?

Reply:
I don't think there's any particular reason, I just wasn't organizing this module around a `pudl_out` object being passed around.